deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -524,15 +524,6 @@ groups:
           type: "ceph_default"
   - name: "pools"
     rules:
-      - alert: "CephPoolGrowthWarning"
```

Comment:
Why does this warning not apply to Rook? Seems like an useful alert regarding the pool storage growth

Reply:
Yes, it should be a useful alert, but when it was enabled last year we had several users complain about it on this issue [here](https://github.com/rook/rook/issues/10070#issuecomment-1099455108). The alert condition hasn't changed, so I'm suggesting we remove it again until we can see a change that should fix it.